PlayerSkeleton.java

```diff
@@ -1,77 +1,83 @@
+import java.util.Arrays;
+
 public class PlayerSkeleton {
+    public static final boolean trackMoveSelection = false; // set true to turn logging on for move selection
 
-	//implement this function to have a working system
-	public int pickMove(State s, int[][] legalMoves) {
-		return pickMove(s, legalMoves, null);
-	}
+    //implement this function to have a working system
+    public int pickMove(State s, int[][] legalMoves) {
+        //		double[] weights = {-0.4944712922445228, 0.14814799954876456, -0.9250354407343329, -1.210228068000284, -0.6363751749245313, -0.3071493143829803,
+//				0.4062421734216026, 0.4053785630482645, 0.7203562102658518, 0.6458317403226688, 0.33541924448498417,
+//				0.156339060819642, 1.2098194382476808, -0.12143020086913736, 1.5266470763709517, 0.5760539680303807,
+//				0.11214199088469634, -0.2072277305271486, 1.2884998585410108, 0.9616177885473564, 0.13438465406024958, 1.1912072595739716};
+//		double[] weights = {-4.500158825082766,3.4181268101392694,-3.2178882868487753,-9.348695305445199,-7.899265427351652,-3.3855972247263626};
+        return pickMove(s, legalMoves, null);
```

Comment:
If you want to play with PlayerSkeleton with your own defined sets of weights (be it from learning algorithms or from online), please input your weights array here and change the return to `pickMove(s, legalMoves, weights);`